### ISSUE TITLE
Outdated `globby` dependency

### ISSUE THREAD

**Issue by medikoo** [serverless/serverless#7689]
https://github.com/serverless/serverless/blob/3a8014089033887621de32b57eed4511616b635f/package.json#L181

We're relying on v6, while we can [safely](https://github.com/archiverjs/node-archiver/blob/master/CHANGELOG.md) (without breaking support for Node.js v6) upgrade to v9.

It needs to be done with care, with full understanding of changes that were made to globby defaults and options handling between v6 and v9 ([changelog](https://github.com/sindresorhus/globby/releases))

PR's welcome!

**Comment by vietduc01100001** [comment]
Hi!
I tried upgrading globby to v9.2.0. There were 2 breaking changes in [v7.0.0](https://github.com/sindresorhus/globby/releases/tag/v7.0.0) that affects this framework:

- `nodir` is now `true` by default, which is fine because `nodir` is explicitly set to `true` here.
https://github.com/serverless/serverless/blob/2b9f63e3329d6e28c0a87d58658b0afde557053e/lib/plugins/package/lib/packageService.js#L249-L254
- `expandDirectories` was added and is `true` by default. I set it to `false`.

All tests passed. So can I make a PR?
